tutorials/R/download-explore/NEON-download-explore.Rmd

```diff
@@ -130,67 +127,125 @@ called **stackedFiles**, which should contain five files:
+packages generally include additional information about data 
+quality, such as chemical standards and quality flags. Not every 
+data product has an expanded package; if the expanded package is 
+requested but there isn't one, the basic package will be 
+downloaded.
+* `avg`: defaults to "all", to download all data; or the 
 number of minutes in the averaging interval. See example below; 
-only applicable to IS data
+only applicable to IS data.
 * `savepath`: the file path you want to download to; defaults to the 
-working directory
+working directory.
 * `check.size`: T or F: should the function pause before downloading 
 data and warn you about the size of your download? Defaults to T; if 
 you are using this function within a script or batch process you 
 will want to set it to F.
+* `nCores`: Number of cores to use for parallel processing. Defaults 
+to 1, i.e. no parallelization.
+* `forceParallel`: If the data volume to be processed does not meet 
+minimum requirements to run in parallel, this overrides.
+
+The `dpID` is the data product identifier of the data you want to 
+download. The DPID can be found on the 
+<a href="http://data.neonscience.org/data-products/explore" target="_blank">
+Explore Data Products page</a>.
+It will be in the form DP#.#####.###
 
 Here, we'll download woody vegetation structure data from 
 Wind River Experimental Forest (WREF).
 
 ```{r run-zipsByProduct, results="hide", message=FALSE, warning=FALSE}
 
-zipsByProduct(dpID="DP1.10098.001", site="WREF", 
-              package="expanded", check.size=T, savepath="~/Downloads")
+veg_str <- loadByProduct(dpID="DP1.10098.001", site="WREF", 
+              package="expanded", check.size=T)
 
 ```
 
-In the file location for your download, you should now have a 
-folder named filesToStack10098. Use `stackByTable()` to stack 
-the files in this folder, as above:
+The object returned by `loadByProduct()` is a named list of data 
+frames. To work with each of them, select them from the list 
+using the `$` operator.
 
-```{r zips-to-stack, results="hide", message=FALSE, warning=FALSE}
+```{r loadBy-list, eval=F}
 
-stackByTable(filepath="~/Downloads/filesToStack10098")
+names(veg_str)
+View(veg_str$vst_perplotperyear)
 
 ```
 
-The **filesToStack10098** folder should now contain a **stackedFiles** 
-folder with seven files:
+If you prefer to extract each table from the list and work 
+with it as an independent object, you can use the 
+`list2env()` function:
 
-* vst_shrubgroup.csv
-* vst_perplotperyear.csv
-* vst_mappingandtagging.csv
-* vst_apparentindividual.csv
-* variables.csv
-* validation.csv
-* readme.txt
+``` {r env, eval=F}
 
-We'll look at these files in more detail below.
+list2env(veg_str, .GlobalEnv)
+
+```
+
+For more details about the contents of the data tables and metadata tables, 
+check out the <a href="https://www.neonscience.org/download-explore-neon-data" target="_blank">Download and Explore</a> 
```

Comment:
That's where we are! I think we can just drop these lines

Reply:
Whoops! Hasty copy/paste. I have removed this completely.